tests/framework/extensions/provisioninginput/config.go

```diff
@@ -191,6 +191,19 @@ type NodePools struct {
 	NodeRoles nodepools.NodeRoles `json:"nodeRoles" yaml:"nodeRoles" default:"[]"`
 }
 
+type ETCD struct {
```

Comment:
why are these structs defined twice. One in etcd snapshot and the other here?

Reply:
When working with the `provisioningInput`, we want to be able to have the flexibility to customize how the backups are on the downstream cluster. I separated it into `snapshotInput` so that when working directly with the snapshot test suite, only that needs to be defined in the test suite.